indico/modules/events/tracks/templates/display.html

```diff
@@ -24,9 +24,15 @@
     <ul class="programme">
         {% for track in event.tracks %}
             <li>
-                <span class="title">
-                    {{- track.title -}}
-                </span>
+                {% if track.can_convene(user) or track.can_review_abstracts(session.user) %}
```

Comment:
If you login as someone who's not an admin, manager or global convener/reviewer I'm quite sure you are now sending at least one extra query for each track in the event.